quantaq/endpoints/devices/__init__.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, client) -> None:
     def list(self, **kwargs) -> list:
         """Return all (available) devices.
 
+        :param int org_id: Filter to devices owned by given org.
+        :param int network_id: Filter to devices owned by given network.
```

Comment:
Note that this doesn't actually involve any code, just `kwargs`. Added it to address stuff like: https://github.com/quant-aq/py-quantaq/issues/14.